tests/jballerina-unit-test/src/test/resources/test-src/types/never/never-type-negative.bal

```diff
@@ -260,3 +260,10 @@ function testNeverRestFieldType() {
     record {||} a3 = {};
     record {|int x;never...; |} copy4 = a3;
 }
+
+never N = check error("Error"); // error
```

Comment:
Shall we add a reference type example aslo?

```ballerina
type FOO never;
Foo N = check error("Error"); //error
```

Reply:
Added